uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar-month.component.js

```diff
@@ -56,7 +59,7 @@ define([
      *
      * @param {LeaveRequestInstance} leaveRequest
      */
-    function addLeaveRequest (__, leaveRequest) {
```

Comment:
Does this mean that pubSub subscription does not pass event as a first param by default? (not a request change)

Reply:
No, pubSub doesn't pass the event:

https://github.com/civicrm/civihr/blob/staging/org.civicrm.reqangular/src/common/services/pub-sub.js#L51